File: Solutions/006/Solution.java
public class Solution{
    public static void main(String[] args){
        int sum = 0;
        int sumofsquares = 0;
        for(int i = 1; i <= 100; i++){
            sum += i;
            sumofsquares += i*i;
        }
        int squareofsum = sum * sum;
        System.out.println("Ans: " + (squareofsum - sumofsquares));
    }
}

File: Solutions/006/Solution.c
#include <stdio.h>

int main(){
    int sumofsquares = 0, sum = 0, i;
    for(i = 1; i <= 100; i++){
        sum += i;
        sumofsquares += i*i;
    }
    int squareofsum = sum * sum;
    printf("Ans: %d\n", squareofsum - sumofsquares);
}